sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientClientLiveTests.cs

```diff
@@ -36,7 +36,8 @@ private LogsQueryClient CreateClient()
                 TestEnvironment.Credential,
                 InstrumentClientOptions(new LogsQueryClientOptions()
                 {
-                    Diagnostics = { IsLoggingContentEnabled = true }
+                    Diagnostics = { IsLoggingContentEnabled = true },
+                    Audience = TestEnvironment.GetAudience()
```

Comment:
I think it makes sense just to make two separate `GetAudience` functions in this case (i.e. `GetLogsAudience` and `GetMetricsAudience`). 
```suggestion
                    Audience = TestEnvironment.GetLogsAudience()
```